kinematics_solver:
  max_iter: 10000
  min_error: 0.001